README.md

```diff
@@ -487,6 +487,23 @@ Seeds, in addition to the above materialization modifiers, have the following op
       <project-name>:
         +use_fastload: true
     ```
+#### Snapshots
+Snapshots uses the HASHROW function of the Teradata database to generate a unique hash value for the 'dbt_scd_id' column. If you want to use your own hash UDF, there is a configuration option in the snapshot model called 'snapshot_hash_udf', which defaults to HASHROW. You can provide a value like <database_name.hash_udf_name>. If only hash_udf_name is provided, it uses the same schema as the model runs.
+
+for e.g. :
+  snapshots/snapshot_example.yml

```

Comment:
The below example seems to be the .sql file, not .yml
Can you please alter that?
One more question, will users be able to provide snapshot_hash_udf configuration from .yml files as well ? for example schema.yml or dbt_project.yml?

Reply:
Corrected. Changed to .sql
No, user can only can use this configuration in the snapshot model sql.